###  Homework 5/

#### Задача 1.

Когда используется критерий Стьюдента, а когда Z –критерий?

Z-критерий используется при соблюдениии нормальности выборки, известна $\sigma$ генеральной совокупности.

t-критерий, или критерий Стьюдента, используется в случае, если $\sigma$ генеральной совокупности неизвестна, соблюдены нормальность выборки и равенство дисперсий в сравниваемых группах.

#### Задача 2

Проведите тест гипотезы. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 кв. мм.

$\sigma$ = 2 кв.мм.

$\alpha$ = 0.05

n = 100

$X_{ср}$ = 17.5 мм

$\mu$ = 17 мм

$$Z_н = \frac{X_{ср}-\mu}{\sigma/\sqrt{n}}$$

In [25]:
z = (17.5-17)/(2/10)
z

2.5

Так как расчетный z получился больше табличного (1.645), то отвергаем нулевую гипотезу и принимаем альтернативную.

#### Задача 3

Проведите тест гипотезы. 

Продавец утверждает, что средний вес пачки печенья составляет 200 г.

Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190.

Известно, что их веса распределены нормально.

Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%? (Провести двусторонний тест.)

$\sigma_н$ = 4.225 гр

$\alpha$ = 0.01

n = 10

$X_{ср}$ = 198.5 гр

$\mu$ = 200 гр

$$t_н = \frac{X_{ср}-\mu}{\sigma_н/\sqrt{n}}$$

In [26]:
import numpy as np
cookie = [202, 203, 199, 197, 195, 201, 200, 204, 194, 190]
sigma = np.std(cookie,ddof=1)
sigma


4.453463071962462

In [27]:
avg = np.mean(cookie)
avg

198.5

In [28]:
t_n = (avg-200)/(sigma/np.sqrt(len(cookie)))
t_n

-1.0651074037450896

Значение для t-критерия по таблице равен 3.25, а рассчетный равен -1.065. Таким образом принимаем нулевую гипотезу.

#### Задача 4.

Задачу 4 решать с помощью функции. 

Есть ли статистически значимые различия в росте дочерей?

Рост матерей 172, 177, 158, 170, 178,175, 164, 160, 169, 165

Рост взрослых дочерей: 173, 175, 162, 174, 175, 168, 155, 170, 160

In [29]:
mothers = [172, 177, 158, 170, 178,175, 164, 160, 169]
daughter = [173, 175, 162, 174, 175, 168, 155, 170, 160]
stats.ttest_rel(mothers,daughter)

Ttest_relResult(statistic=0.559522990335608, pvalue=0.5911212354055175)

Для статистической значимости $\alpha$ = 0.05 статистически значимых различий нет.